src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionController.php

```diff
@@ -108,7 +108,9 @@ protected function getTemplate()
         return '@Twig/Exception/'.($this->debug ? 'exception' : 'error').'.html.twig';
     }
 
-    // to be removed when the minimum required version of Twig is >= 2.0
+    /**
+     * To be removed when the minimum required version of Twig is >= 2.0.
```

Comment:
The fact it is to be removed is useless for people consuming the code. This would just display them internal reminders of the devs